servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -46,19 +46,19 @@ The embedded RavenDB will use additional RAM to improve indexing performance. Du
 
 ### Message size / MaxBodySizeToStore
 
-In general, the smaller the messages, the faster ServiceControl will be able to process audit records. Thus, [put events on a diet](https://particular.net/blog/putting-your-events-on-a-diet) if optimizing for performance. For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
+In general, the smaller the messages, the faster ServiceControl will process audit records (see [this blog post](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
 
-In addition, for audit messages, lower the [`ServiceControl/MaxBodySizeToStore`](/servicecontrol/creating-config-file.md#performance-tuning-servicecontrolmaxbodysizetostore) setting to skip storage of larger audit messages. This setting will only reduce load if a non-binary [serialization](/nservicebus/serialization/) is used.
+For audit messages, lower the [`ServiceControl/MaxBodySizeToStore`](/servicecontrol/creating-config-file.md#performance-tuning-servicecontrolmaxbodysizetostore) setting to skip storage of larger audit messages. This setting will only reduce load if non-binary [serialization](/nservicebus/serialization/) is used.
 
-WARNING: When using ServiceInsight, the message body will not be viewable for messages that exceed the `ServiceControl/MaxBodySizeToStore` limit.
+WARNING: When using ServiceInsight, the message body is not viewable for messages that exceed the `ServiceControl/MaxBodySizeToStore` limit.
 
 ### Separate disks for database and index files
 
 Besides using a dedicated disk for the ServiceControl [database paths](/servicecontrol/creating-config-file.md#host-settings-servicecontroldbpath), it's possible to store the embedded database index files on a separate disk. Use the [`Raven/IndexStoragePath`](/servicecontrol/creating-config-file.md#host-settings-ravenindexstoragepath) setting to change the index storage location.
```

Comment:
I'm not sure if this actually still works with Raven 5.

Reply:
As discussed, these docs are due for review soon so we'll leave this for then.